### DISCUSSION TITLE
When using the `JavascriptObjectRepository.Register` function in WinForms to facilitate mutual calls between JavaScript and C#, only lowercase function names are effective.

### DISCUSSION THREAD

**Discussion by JDDKCN** [cefsharp/CefSharp#4684] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

120.1.80

### Operating System

Windows 11

### Architecture

x64

### .Net Version

.NET Framework 4.7.2

### Implementation

WinForms

### Reproduction Steps

- CSharp Code
```CSharp
        public void CefInitTest()
        {
            browser = new ChromiumWebBrowser("http://127.0.0.1:8080");
            CefSharpSettings.WcfEnabled = true;
            browser.JavascriptObjectRepository.Settings.LegacyBindingEnabled = true;
            browser.JavascriptObjectRepository.Register("BackendService", new BackendService(), isAsync: false, options: BindingOptions.DefaultBinder);
            panel1.Controls.Add(browser);
        }

    public class BackendService
    {
        public void OK(string msg,string title)
        {
            MessageBox.Show(msg,title);
        }
    }
```

- JS Code
```JavaScript
function test() {
    BackendService.ok('Message', 'Title');
}
```

### Expected behavior

When the JavaScript code is triggered, the WinForms window should pop up a MessageBox.

### Actual behavior

- [Web Console] Uncaught ReferenceError: BackendService is not defined
- [VS Console] Uncaught TypeError: window.BackendService.OK is not a function

### Regression?

I don't know.

### Known Workarounds

Change the function names to lowercase; uppercase characters are not allowed.

### Does this problem also occur in the CEF Sample Application

Yes using WinForms command line args

**Comment by Goldenringz** [comment]
Beautiful !

**Comment by amaitland** [comment]
Converting to discussion as this is not a bug. Naming is done via convention. The default convention simply converts the first character to lowercase.

```
OK -> oK
Ok -> ok
```

If you wish to use a different convention then you can set [IJavascriptObjectRepository.NameConverter.](https://cefsharp.github.io/api/118.6.x/html/P_CefSharp_IJavascriptObjectRepository_NameConverter.htm) to your own instance of `IJavascriptNameConverter`

In future please use discussions to ask support questions.
